extensions/git/src/git.ts

```diff
@@ -1285,6 +1285,26 @@ export class Repository {
 		await this.run(args);
 	}
 
+	async deleteTag(name: string): Promise<void> {
+		let args = ['tag', '-d', name];
+		await this.run(args);
+	}
+
+	async getTags(): Promise<Tag[]> {
```

Comment:
This is not needed. We already have a list of tags in `repository.refs`. I see that this brings in the tag message, but let's assume the user's not really interested in that at the point that they wish to delete a tag.